Scripts/DCS-BIOS/lib/modules/Module.lua

```diff
@@ -105,12 +105,12 @@ end
 --- Defines a gauge from floating-point data with limits
 --- @param identifier string the unique identifier for the control
 --- @param arg_number integer the dcs argument number
+--- @param max_value integer maximum possible value
 --- @param limits number[] a length-2 array with the lower and upper bounds of the data as used in dcs
 --- @param category string the category in which the control should appear
 --- @param description string additional information about the control
 --- @return Control control the control which was added to the module
-function Module:defineFloat(identifier, arg_number, limits, category, description)
-	local max_value = 65535
+function Module:defineFloatValue(identifier, arg_number, max_value, limits, category, description)
```

Comment:
You can add `--- @private` to the top of the documentation to indicate this function should not be used outside of this file